src/components/ChannelPreview/ChannelPreviewStatus.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { useTheme } from '../../contexts';
+import { StyleSheet, Text, View } from 'react-native';
+import type {
+  DefaultAttachmentType,
+  DefaultChannelType,
+  DefaultCommandType,
+  DefaultEventType,
+  DefaultMessageType,
+  DefaultReactionType,
+  DefaultUserType,
+  UnknownType,
+} from '../../types/types';
+import { Check, CheckAll } from '../..//icons';
```

Comment:
Should re-order, space-out, and I think line 14 will break

Reply:
Sorry, was using vs code auto-imports 😬